pymc/tests/test_distributions_timeseries.py

```diff
@@ -15,17 +15,103 @@
 import numpy as np
 import pytest
 
+from scipy import stats
+
+import pymc as pm
+
 from pymc.aesaraf import floatX
 from pymc.distributions.continuous import Flat, Normal
-from pymc.distributions.timeseries import AR, AR1, GARCH11, EulerMaruyama
+from pymc.distributions.timeseries import (
+    AR,
+    AR1,
+    GARCH11,
+    EulerMaruyama,
+    GaussianRandomWalk,
+    gaussianrandomwalk,
+)
 from pymc.model import Model
 from pymc.sampling import sample, sample_posterior_predictive
 from pymc.tests.helpers import select_by_precision
 
-# pytestmark = pytest.mark.usefixtures("seeded_test")
-pytestmark = pytest.mark.xfail(reason="Timeseries not refactored")
+
+class TestGaussianRandomWalk:
```

Comment:
I would suggest reusing the test classes we already have for RandomVariable Ops: https://github.com/pymc-devs/pymc/blob/c892317aba0663df0d250c1813b2ca2fe3460774/pymc/tests/test_distributions_random.py#L263

And for logp: https://github.com/pymc-devs/pymc/blob/c892317aba0663df0d250c1813b2ca2fe3460774/pymc/tests/test_distributions.py#L626

This will make sure we test this distribution at least as thoroughly as we do other distributions, and also that when we  extend those test classes in the future, this benefits automatically as well: